test/conftest.py

```diff
@@ -35,6 +36,8 @@
 ERROR_LOGIN_REQUIRED = 'Please log in to access this page.'
 ERROR_INVALID_CHOICE = 'Not a valid choice'
 
+config.TRACKER_ISSUES_PER_PAGE = 25
```

Comment:
why do we need to influence the config module here?

Reply:
After debugging I can provide some python / py.test gotchas.
If you try to monkeypatch a module after it was loaded it won't work. If we move this line into test_index.py the test would fail on `make test` but run fine on `pytest test/test_index.py` - If we wan't to monkeypatch it, we might need to swich to a generic `import config` in `views/index.py` and switch from `TRACKER_ISSUES_PER_PAGE` to `config.TRACKER_ISSUES_PER_PAGE`. This however would change the main app. I'd prefer living with the setting in the conftest.py

A completely different solution would be to have an extra XX-test.conf similar to 00-default.conf that is only loaded for tests where we could provide default constants without fearing those being changed.